framework/tests/src/tests/ValueUncheckedDefaultsTest.java

```diff
@@ -0,0 +1,29 @@
+package tests;
+
+import java.io.File;
+import java.util.List;
+import org.checkerframework.common.value.ValueChecker;
+import org.checkerframework.framework.test.CheckerFrameworkPerDirectoryTest;
+import org.junit.runners.Parameterized.Parameters;
+
+/** Tests unchecked code defaults for the constant value propagation type system. */
+public class ValueUncheckedDefaultsTest extends CheckerFrameworkPerDirectoryTest {
+
+    public ValueUncheckedDefaultsTest(List<File> testFiles) {
+        super(
+                testFiles,
+                ValueChecker.class,
+                "value",
+                "-Anomsgtext",
+                "-Astubs=statically-executable.astub",
+                "-AuseDefaultsForUncheckedCode=btyecode",
```

Comment:
Hi @smillst , we've just been looking at this code and notice that `btyecode` is not spelled correctly.
Should there be an error message for incorrectly spelled arguments?

Reply:
Yes, there should be.  Do you want to implement that, or are you asking Suzanne to do so?
Thanks for noticing this.

Reply:
I assume that with the invalid flag this whole test case doesn't do what was intended.
So it would be best for @smillst to have a look through the test and in the process implement an error for invalid keys.